core/lib/engine_http.js

```diff
@@ -720,7 +720,11 @@ HttpEngine.prototype._handleResponse = function (
     if (rawCookies) {
       context._enableCookieJar = true;
       rawCookies.forEach(function (cookieString) {
-        context._jar.setCookieSync(cookieString, url);
+        try {
+          context._jar.setCookieSync(cookieString, url);
+        } catch (err) {
+          ee.emit('error', 'Could not parse cookieString from response header, skipping it ' + cookieString);
```

Comment:
The messages from these `error` events go right into the metric report, so it's best to keep it short. How about something like `cookie_parse_error_invalid_cookie`?

For debugging we can log the current message with `debug()` instead.

Reply:
Done, used uppercase for the error message for consistency.
One detail is that we can get N error messages for one call (as there may be N invalid Set-Cookie headers)